src/go/rdctl/pkg/shutdown/shutdown.go

```diff
@@ -201,20 +221,36 @@ func runCommandIgnoreOutput(cmd *exec.Cmd) error {
 	return cmd.Run()
 }
 
-func stopLima() error {
+func stopLima(ctx context.Context) error {
 	return runCommandIgnoreOutput(exec.Command(limaCtlPath, "stop", "0"))
```

Comment:
We are injecting `ctx` we don't even use it, why can't we pass it to `exec.CommandContext(ctx,....`